packages/client/src/__tests__/integration/happy/composites-mongo/update/list.ts

```diff
@@ -242,12 +242,108 @@ describeIf(!process.env.TEST_SKIP_MONGODB)('update > list', () => {
   /**
    * Simple updateMany
    */
-  test.skip('updateMany', async () => {})
+  test('updateMany', async () => {
+    const comment = await prisma.commentRequiredList.update({
+      where: { id },
+      data: {
+        contents: {
+          updateMany: {
+            data: {
+              upvotes: [{ userId: 'Another Comment', vote: true }],
+            },
+            where: {
+              upvotes: {
+                isEmpty: true,
+              },
+            },
+          },
+        },
+      },
+    })
+
+    expect(comment).toMatchInlineSnapshot(`
+      Object {
+        contents: Array [
+          Object {
+            text: Goodbye World,
+            upvotes: Array [
+              Object {
+                userId: 11,
+                vote: false,
+              },
+            ],
+          },
+          Object {
+            text: Hello World,
+            upvotes: Array [
+              Object {
+                userId: 10,
+                vote: true,
+              },
+            ],
+          },
+          Object {
+            text: Hello World,
+            upvotes: Array [
+              Object {
+                userId: Another Comment,
```

Comment:
Nit: it's a bit weird to mix a number and a string here for a userId.

I see the type is `userId String` in the schema

Reply:
Nice catch, thanks! These will be caught when we have automated typechecks (coming soon).